app/test_app/functional/me_api/test_me_credit_transfer.py

```diff
@@ -13,3 +15,41 @@ def test_get_me_credit_transfer_api(test_client, create_credit_transfer, create_
                                content_type='application/json', follow_redirects=True)
     assert response.status_code == 201
     assert response.json['data']['credit_transfers'][0]['id'] is create_credit_transfer.id
+
+
+def test_invalid_create_me_credit_transfer_api(
```

Comment:
adding a test for a _valid_ credit transfer could be worth it also

Reply:
that's the one above